pkg/apis/internal.admin.acorn.io/v1/baseresources_test.go

```diff
@@ -113,107 +120,73 @@ func TestBaseResourcesRemove(t *testing.T) {
-			incoming: BaseResources{
-				Apps:   1,
-				Memory: resource.MustParse("1Mi"),
-			},
+			name:     "remove from empty BaseResources resources",
+			current:  BaseResources{},
+			incoming: BaseResources{Apps: 1},
 			expected: BaseResources{},
 		},
 		{
-			name: "remove from existing BaseResources resources",
-			current: BaseResources{
-				Apps:   1,
-				Memory: resource.MustParse("1Mi"),
-			},
-			incoming: BaseResources{
-				Apps:   1,
-				Memory: resource.MustParse("1Mi"),
-			},
+			name:     "remove from existing BaseResources resources",
+			current:  BaseResources{Apps: 1},
+			incoming: BaseResources{Apps: 1},
 			expected: BaseResources{},
 		},
 		{
-			name: "should never get negative values",
-			all:  true,
-			current: BaseResources{
-				Apps:          1,
-				Memory:        resource.MustParse("1Mi"),
-				VolumeStorage: resource.MustParse("1Mi"),
-			},
-			incoming: BaseResources{
-				Apps:          2,
-				Memory:        resource.MustParse("2Mi"),
-				VolumeStorage: resource.MustParse("2Mi"),
-			},
+			name:     "should never get negative values",
+			all:      true,
+			current:  BaseResources{Apps: 1},
+			incoming: BaseResources{Apps: 2},
 			expected: BaseResources{},
 		},
 		{
-			name: "remove persistent resources with all",
-			current: BaseResources{
-				VolumeStorage: resource.MustParse("1Mi"),
-			},
-			incoming: BaseResources{
-				VolumeStorage: resource.MustParse("1Mi"),
-			},
-			all:      true,
-			expected: BaseResources{},
+			name:     "remove where current has a resource specified with unlimited",
+			current:  BaseResources{Apps: Unlimited},
+			incoming: BaseResources{Apps: 1},
+			expected: BaseResources{Apps: Unlimited},
 		},
 		{
-			name: "does not remove persistent resources without all",
-			current: BaseResources{
-				VolumeStorage: resource.MustParse("1Mi"),
-			},
-			incoming: BaseResources{
-				VolumeStorage: resource.MustParse("1Mi"),
-			},
-			expected: BaseResources{
-				VolumeStorage: resource.MustParse("1Mi"),
-			},
+			name:     "remove where incoming has a resource specified with unlimited",
+			current:  BaseResources{Apps: 1},
+			incoming: BaseResources{Apps: Unlimited},
+			expected: BaseResources{Apps: 1},
 		},
 		{
-			name: "remove where current has a resource specified with unlimited",
-			current: BaseResources{
-				Apps:   Unlimited,
-				Memory: UnlimitedQuantity(),
-			},
-			incoming: BaseResources{
-				Apps:   1,
-				Memory: resource.MustParse("1Mi"),
-			},
-			expected: BaseResources{
-				Apps:   Unlimited,
-				Memory: UnlimitedQuantity(),
-			},
+			name:     "remove where current and incoming have a resource specified with unlimited",
+			current:  BaseResources{Apps: Unlimited},
+			incoming: BaseResources{Apps: Unlimited},
+			expected: BaseResources{Apps: Unlimited},
 		},
 		{
-			name: "remove where incoming has a resource specified with unlimited",
+			name: "remove where current and incoming have ComputeClasses and VolumeClasses",
```

Comment:
Can we add a test case here for the suspected issue?

Current has two volume and compute classes, one of each is removed, and the others should still be there.